contracts/src/spaces/facets/membership/MembershipFacet.sol

```diff
@@ -147,8 +150,22 @@ contract MembershipFacet is
       (address, address)
     );
 
-    // allocate protocol and membership fees
-    uint256 membershipPrice = _getMembershipPrice(_totalSupply());
+    uint256 totalSupply = _totalSupply();
+    uint256 membershipPrice;
+
+    uint256 freeAllocation = _getMembershipFreeAllocation();
+    uint256 prepaidSupply = _getPrepaidSupply();
```

Comment:
can you remind me what is the difference between freeAllocation and prepaidSupply?

Reply:
free allocation - value defined by DAO of free memberships you get for a new town
prepaid supply - what a space owner pays for on behalf of their members